### PR TITLE
Create 2022-02-25-ml-in-galaxy.md

### PR DESCRIPTION
**Pull request by anuprulez**

### CODE REVIEW

File: _posts/2022-02-25-ml-in-galaxy.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+tags: [videos]
+title: "Machine learning in Galaxy"
```
Comment (bgruening) [inline _posts/2022-02-25-ml-in-galaxy.md:4]: add yourself as author please, or thank yourself in the last sentence :)

File: _posts/2022-02-25-ml-in-galaxy.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+tags: [videos]
+title: "Machine learning in Galaxy" 
+supporters:
+- galaxy-europe
+- GTN Video Library
```
Comment (bgruening) [inline _posts/2022-02-25-ml-in-galaxy.md:7]: supporters are something like denbi etc.

- denbi
- elixir
- unifreiburg

File: _posts/2022-02-25-ml-in-galaxy.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+- Erasmus+ programme of the European Union
+---
+
+Artificial intelligence (machine learning and deep learning) algorithms are being increasingly applied in several fields of Bioinformatics such as drug-response 
+and protein structure prediction, imputing missing data in single-cell gene expression and many more. To harness the potential and versatility of AI’s algorithms, 
+it becomes of utmost importance to deeply understand its algorithms while applying them to biological datasets for predictive tasks. Galaxy’s ML toolkit provides many popular AI algorithms from Scikit-learn, a popular Python ML library and Keras, a deep learning library based on Google’s Tensorflow, and various other tools for preprocessing and visualization.
```
Comment (bgruening) [inline _posts/2022-02-25-ml-in-galaxy.md:13]: maybe point to your paper here as well?

**Comment by bgruening** [comment]
And this one maybe? https://galaxyproject.eu/posts/2021/06/07/GalaxyML/

**Comment by anuprulez** [comment]
> And this one maybe? https://galaxyproject.eu/posts/2021/06/07/GalaxyML/

ah yes, now added it :)

**Comment by bgruening** [comment]
Cool thanks!

**Pull request merged**